packages/commonwealth/main.ts

```diff
@@ -30,11 +32,11 @@ import { setupCosmosProxies } from './server/util/comsosProxy/setupCosmosProxy';
 import setupIpfsProxy from './server/util/ipfsProxy';
 import ViewCountCache from './server/util/viewCountCache';
 
-// set up express async error handling hack
-require('express-async-errors');
```

Comment:
Why move this to the server files?

Reply:
Good catch, we tried removing it but we can put it back here.